stratum/hal/lib/bcm/bcm_chassis_manager.cc

```diff
@@ -153,6 +153,26 @@ BcmChassisManager::~BcmChassisManager() {
   CleanupInternalState();
 }
 
+/*!
+ * \brief BcmChassisManager::GetModuleNetworkIds method \return (module, netif)
+ * pair related to \param node_id and \param port_id
+ */
+std::pair<uint32, uint32> BcmChassisManager::GetModuleNetworkIds(
+    uint64 node_id, uint32 port_id) const {
+    return node_port_id_to_module_netif.at({node_id, port_id});
+}
+
+/*!
+ * \brief BcmChassisManager::IsNodePortIdRelatedToTAI method check is (node_id,
+ * port_id) pair \param node_port_id related to TAI
+ * \return true if related, false if not related
+ */
+bool BcmChassisManager::IsNodePortIdRelatedToTAI(
+    const std::pair<uint64, uint32>& node_port_id) const {
+  const auto kIterator = node_port_id_to_module_netif.find(node_port_id);
+  return kIterator != node_port_id_to_module_netif.end();
+}
```

Comment:
I think this function is not needed. If a port is not related to TAI the specific function will fail and we can report this then.

Reply:
I've changed the logic a little and removed that function. Please review